.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
   pull_request:
 
 env:
-  nightly: stable
+  stable: stable
```

Comment:
Perhaps "toolchain" was a better name here but no need to hold this PR for this.

Reply:
We don't need it to be an env, as this is probably handled by the `rust-toolchain.toml` but I don't want to mess with it now

Reply:
done